react-training/src/app/components/Topbar/Menu/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import CustomLink from '../../../components/CustomLink';
+import { routes } from '../../../../constants/routes';
+import authActions from '../../../../redux/auth/action';
+
+import styles from './styles.scss';
+
+const Menu = props => (
+  <ul className={styles.list}>
+    <CustomLink tag="Log Out" handleCustomClick={props.logout} />
+    <CustomLink tag="My Account" route={routes.account} />
+    <CustomLink tag="PLAY" route={routes.game[0]} />
+  </ul>
+);
+
+const mapStateToProps = ({ auth }) => auth;
```

Comment:
Are u using `auth`? If u dont use any `mapStateToProps`, just pass null in the first `connect` param
```js
export default connect(
  null,
  mapDispatchToProps
)(Menu);
```